### PR TITLE
glab-issue: add page

### PR DESCRIPTION
**Pull request by reinhart1010**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

See #7554

### CODE REVIEW

File: pages/common/glab-issue.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# glab issue
+
+> Manage Gitlab issues from the command-line.
+> More information: <https://glab.readthedocs.io/en/latest/issue/index.html#synopsis>.
```
Comment (navarroaxel) [inline pages/common/glab-issue.md:4]: I don't see the anchor useful.
```suggestion
> More information: <https://glab.readthedocs.io/en/latest/issue/>.
```

File: pages/common/glab-issue.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Print out the issue:
+
+`glab issue view {{issue_number}}`
+
+- Create a new issue in the web browser:
```
Comment (navarroaxel) [inline pages/common/glab-issue.md:10]:
```suggestion
- Create a new issue in the default web browser:
```

File: pages/common/glab-issue.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- List the last 10 issues with the `bug` label:
+
+`glab issue list --per-page {{10}} --label "{{bug}}"`
+
+- List closed issues made by a specific user:
+
+`glab issue list --closed --author {{username}}`
+
+- Reopen an issue:
```
Comment (navarroaxel) [inline pages/common/glab-issue.md:22]:
```suggestion
- Reopen a specific issue:
```

File: pages/common/glab-issue.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# glab issue
+
+> Manage Gitlab issues from the command-line.
+> More information: <https://glab.readthedocs.io/en/latest/issue/>.
+
+- Print out the issue:
```
Comment (navarroaxel) [inline pages/common/glab-issue.md:6]: I prefer replace the phrasal verb to a simpler English verb:
 ```suggestion
- Display a specific issue:
```
Easier to read for basic English speakers.

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**